mutt/file.c

```diff
@@ -700,6 +703,28 @@ FILE *mutt_file_fopen_full(const char *path, const char *mode, const char *file,
 
   return fp;
 }
+/**
+ * mutt_file_fopen_full - A Wrapper around mutt_file_fopen_full_with_mask with the mask parameter set to 0600
+ */
+FILE *mutt_file_fopen_full(const char *path, const char *mode, const char *file,
+                           int line, const char *func)
+{
+  const mode_t mask = 0600;
+  return mutt_file_fopen_full_with_mask(path, mode, mask, file, line, func);
+}
+
+/**
+ * mutt_file_fopen_full - A wrapper around mutt_file_fopen_full_with_mask with
+ * the mask parameter set to the user's default mode inferred from the umask
+ */
+FILE *mutt_file_fopen_full_masked(const char *path, const char *mode,
+                                  const char *file, int line, const char *func)
+{
+  struct ConfigSet *cs = cs_new(16);
+  NeoMutt = neomutt_new(cs);
```

Comment:
Hmm... That's a surprise!
There can be only one NeoMutt and this isn't it :-)

The `mutt/` library is intended to be self-contained (dependency free).
Any config or global variables need to be passed in as parameters.

Reply:
> I didn't understand the meaning of `fuzz`

Ah, that's a standalone [fuzzing](https://en.wikipedia.org/wiki/Fuzzing) tester.
It creates a fake `NeoMutt` to keep the linked libraries happy.

> I don't understand how to access `NeoMutt`

The global `NeoMutt` object is declared in `core/neomutt.h`, so `#include "core/lib.h"` will get you access.

Reply:
> The global `NeoMutt` object is declared in `core/neomutt.h`, so `#include "core/lib.h"` will get you access.

It is already included in `file.c`...

Reply:
It works :)